src/groups/FBX.tsx

```diff
@@ -0,0 +1,245 @@
+// FBX.tsx
+import * as React from "react";
+import * as THREE from "three";
+import { FBXLoader } from "three/examples/jsm/loaders/FBXLoader";
```

Comment:
I wished they'd expose those utils in a package. 😢

Reply:
Me too